modules/ROOT/pages/dw-core-namespaces.adoc

```diff
@@ -1,11 +1,14 @@
 = Core Namespaces (dw::Core)
 
-
 |===
 | Namespace | Definition | Description
 
 | xsi
-| `xsi =` `+http://www.w3.org/2001/XMLSchema-instance+`
+| `xsi = http://www.w3.org/2001/XMLSchema-instance`
```

Comment:
NOTE: manually change `xsi = http://www.w3.org/2001/XMLSchema-instance` to `+xsi = http://www.w3.org/2001/XMLSchema-instance+`

TODO: DW Jira to see if that can be changed in source without issues